packages/shared/akamai-edgeworker-sdk/__tests__/featureStore/cacheableStore.test.ts

```diff
@@ -0,0 +1,96 @@
+import { EdgeProvider } from '../../src/featureStore';
+import CacheableStoreProvider from '../../src/featureStore/cacheableStoreProvider';
+import * as testData from '../testData.json';
+
+describe('CacheableStoreProvider', () => {
+  const mockEdgeProvider: EdgeProvider = {
+    get: jest.fn(),
+  };
+  const mockGet = mockEdgeProvider.get as jest.Mock;
+
+  beforeEach(() => {
+    mockGet.mockImplementation(() => Promise.resolve(JSON.stringify(testData)));
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('without cache TTL', () => {
+    it('caches initial request', async () => {
+      const cacheProvider = new CacheableStoreProvider(mockEdgeProvider, 'rootKey');
+      await cacheProvider.get('rootKey');
+      await cacheProvider.get('rootKey');
+      expect(mockGet).toHaveBeenCalledTimes(1);
+    });
+
+    it('can force a refresh', async () => {
+      const cacheProvider = new CacheableStoreProvider(mockEdgeProvider, 'rootKey');
+      await cacheProvider.get('rootKey');
+      await cacheProvider.get('rootKey');
+      expect(mockGet).toHaveBeenCalledTimes(1);
+
+      await cacheProvider.prefetchPayloadFromOriginStore();
+      await cacheProvider.get('rootKey');
+      expect(mockGet).toHaveBeenCalledTimes(2);
+    });
+  });
+
+  describe('with infinite cache ttl', () => {
+    it('caches initial request', async () => {
+      const cacheProvider = new CacheableStoreProvider(mockEdgeProvider, 'rootKey', 0);
+      await cacheProvider.get('rootKey');
+      await cacheProvider.get('rootKey');
+      expect(mockGet).toHaveBeenCalledTimes(1);
+    });
+
+    it('prefetch does not reset', async () => {
```

Comment:
```suggestion
    it('does not reset on prefetch', async () => {
```

I know I have not gone through and refactored all the tests in the project, but I try to write these such that `it` is part of what you read.